android-demos/GeotriggerMonitoring/GeotriggerMonitoringDemo-WithGeotriggers/settings.gradle

```diff
@@ -0,0 +1,11 @@
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        jcenter() // Warning: this repository is going to shut down soon
```

Comment:
do you still need jcenter?  As you've added mavenCentral you probably don't need it